packages/design-system/src/components/N8nButton/Button.stories.ts

```diff
@@ -0,0 +1,192 @@
+/* tslint:disable:variable-name */
+import N8nButton from './Button.vue';
+import { action } from '@storybook/addon-actions';
+import { StoryFn } from "@storybook/vue";
+
+export default {
+	title: 'Atoms/Button',
+	component: N8nButton,
+	argTypes: {
+		label: {
+			control: 'text',
+		},
+		title: {
+			control: 'text',
+		},
+		type: {
+			control: 'select',
+			options: ['primary', 'outline', 'light', 'text', 'tertiary'],
```

Comment:
need to update the types here to match new ones

Reply:
Good catch. Updated!